.gitignore

```diff
@@ -0,0 +1,188 @@
+schema/
+src/main/java/th/or/etda/
```

Comment:
This means we are .gitignore the entire namespace. Is this intentional?

Also normally we would generate code into "src/generated/java" for a clean separation between generated and src. but let's fix that in another PR after this is merged.

Reply:
oppss!!! "src/generated/java" is the proper name, just thinking the proper name once working on this PR somehow cannot haha